setup/hyphenation.py

```diff
@@ -83,7 +83,7 @@ class Hyphenation(ReVendor):
     description = 'Download the hyphenation dictionaries'
     NAME = 'hyphenation'
     TAR_NAME = 'hyphenation dictionaries'
-    VERSION = 'master'
+    VERSION = 'libreoffice-7.6.5.2'
```

Comment:
This change is bad and should not be done.

```
$ python setup.py hyphenation --help
[...]
  --path-to-hyphenation=PATH_TO_HYPHENATION
                        Path to the extracted hyphenation dictionaries source
  --hyphenation-url=HYPHENATION_URL
                        URL to hyphenation dictionaries source archive in
                        tar.gz format
```

I implemented this in https://github.com/kovidgoyal/calibre/pull/1074 for... non-network builds, as it happens. My motivation was actually to produce a recipe that built calibre from git master, not from git tags.

Reply:
You are right, this change is unnecessary.